### DISCUSSION TITLE
Wordpress plugin

### DISCUSSION THREAD

**Discussion by aheissenberger** [zitadel/zitadel#6713] [Q&A]
Which plugin is recommended to connect a Wordpress website with zitadel?
Is it possible to use any OAuth 2.0 client plugin or are there any additional requirements?

**Comment by hifabienne** [comment]
Hei @aheissenberger 
I am not really familiar with wordpress.
If there is a plugin that supports a generic OIDC provider I would recommend using this

**Comment by mffap** [comment]
Hey @aheissenberger have a look at this PR, that might help. But make sure to checkout the comments regarding claims/roles.

https://github.com/oidc-wp/openid-connect-generic/issues/452

**Comment by misbahjabeen931** [comment]
https://w2beginner.com/wordpress-course-in-rawalpindi/

This WordPress training course will teach you how to  a professional WordPress website for yourself and your clients.

**Comment by misbahjabeen931** [comment]
https://w2beginner.com/wordpress-course-in-rawalpindi/

get learn wordpress

**Comment by MShahbaz223** [comment]
The suggested plugin for connecting a WordPress website to Zitadel is Zitadel for WordPress. It's user-friendly and offers a range of qualities such as SSO, user provisioning, position mapping, group mapping, and user profile synchronization.
For those aiming for more flexibility, there's the option to utilize the OAuth 2.0 Client plugin to connect with Zitadel.

**Reply by aheissenberger** [reply]
Where can I find the plugin "Zitadel for WordPress"? It is not listed at Wordpress https://wordpress.org/plugins/ and not found by Google Search.
